changelogs/current.yaml

```diff
@@ -90,6 +90,10 @@ minor_behavior_changes:
   change: |
     Filter metadata containing ext proc stats has been moved from ext-proc-logging-info to a namespace corresponding
     to the name of the ext_proc filter.
+- area: upstream
```

Comment:
This shouldn't be part of `minor behavior change`, this should be part of new features